### PR TITLE
Remove mock data and all related utilities (no JIRA)

### PR DESCRIPTION
**Pull request by MattPutnam**
Mock data hasn't been used in probably over a year. We haven't maintained the mock responses on endpoints, and it's completely unusable. Let's just get rid of it.

- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [x] **Product Owner** sign off
- [x] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [x] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [x] **Submitter**: Delete branch after merge
- [x] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [x] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/cljs/main/broadfcui/page/workspace/create.cljs [hunk 1]
```diff
@@ -103,7 +103,7 @@
                                                 (:selected-groups @state))}]
          (swap! state assoc :creating-ws true)
          (endpoints/call-ajax-orch
-          {:endpoint (endpoints/create-workspace project name)
+          {:endpoint endpoints/create-workspace
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/page/workspace/create.cljs:106]: This is the annoyance that led to this.

File: src/cljs/main/broadfcui/endpoints.cljs [hunk 2]
```diff
@@ -525,65 +235,10 @@
...
-                :entityType (rand-nth ["Task" "Workflow"])}
-       :namespace (rand-nth ["Broad" "nci" "public" "ISB"])
-       :name (str "Configuration " (inc i))
-       :snapshotId (rand-int 100)
-       :synopsis (str (rand-nth ["variant caller synopsis", "gene analyzer synopsis", "mutect synopsis"]) " " (inc i))
-       :documentation (str "Documentation for config " (inc i))
-       :createDate (rand-recent-time)
-       :payload "task wc {File in_file command { cat ${in_file} | wc -l } output { Int count = read_int(stdout()) }}\n"
-       :entityType "Task"})
-    (range (rand-int 50)))})
```
Comment (davidangb) [inline src/cljs/main/broadfcui/endpoints.cljs:551]: this endpoint was removed entirely, intentional?

File: src/cljs/main/broadfcui/endpoints.cljs [hunk 3]
```diff
@@ -525,65 +235,10 @@
...
-    :namespace namespace
-    :name name
-    :snapshotId snapshot-id
-    :synopsis (rand-nth ["variant caller synopsis", "gene analyzer synopsis", "mutect synopsis"])
-    :documentation (str "Documentation for config")
-    :createDate (rand-recent-time)
-    :payload "task wc {File in_file command { cat ${in_file} | wc -l } output { Int count = read_int(stdout()) }}\n"
-    :entityType "Task"}})
-
-(defn copy-method-config-to-workspace [workspace-id]
```
Comment (davidangb) [inline src/cljs/main/broadfcui/endpoints.cljs:576]: this endpoint was removed entirely, intentional?

**Review by davidangb — APPROVED** [review]
👍 code looks good, did not test. Two "are you sure?" questions inline.

**Comment by MattPutnam** [comment]
Yeah, those two endpoints were unused.

**Review by rushtong — APPROVED** [review]
👍 Looks good to me. Missing endpoint questions answered!

**Review by zarsky-broad — APPROVED** [review]
Don't forget to get rid of the bottom right toggle link, otherwise LGTM

**Pull request merged**